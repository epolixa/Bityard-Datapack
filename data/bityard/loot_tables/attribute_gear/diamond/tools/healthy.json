{
  "pools": [
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "minecraft:diamond_sword",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Diamond Sword\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -2.4,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 6.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:diamond_pickaxe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Diamond Pickaxe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -2.8,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 4.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:diamond_axe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Diamond Axe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -3.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 8.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:diamond_shovel",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Diamond Shovel\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -3.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 4.5,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:diamond_hoe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Diamond Hoe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        }
      ]
    }
  ]
}