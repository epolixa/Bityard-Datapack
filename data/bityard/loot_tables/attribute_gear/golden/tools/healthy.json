{
  "pools": [
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "minecraft:golden_sword",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Golden Sword\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -2.4,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 3.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:golden_pickaxe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Golden Pickaxe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -2.8,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 1.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:golden_axe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Golden Axe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -3.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 6.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:golden_shovel",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Golden Shovel\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -3.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 1.5,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:golden_hoe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Golden Hoe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -3.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        }
      ]
    }
  ]
}